--- pkg/harfbuzz/buffer.zig
+++ pkg/harfbuzz/buffer.zig
@@ -261,6 +261,13 @@
 /// Data type for holding HarfBuzz's clustering behavior options. The cluster
 /// level dictates one aspect of how HarfBuzz will treat non-base characters
 /// during shaping.
+const hb_buffer_cluster_level_graphemes = if (@hasDecl(
+    c,
+    "HB_BUFFER_CLUSTER_LEVEL_GRAPHEMES",
+))
+    c.HB_BUFFER_CLUSTER_LEVEL_GRAPHEMES
+else
+    3;
 pub const ClusterLevel = enum(u2) {
     /// In `monotone_graphemes`, non-base characters are merged into the
     /// cluster of the base character that precedes them. There is also cluster
@@ -289,7 +296,7 @@
     /// is not forced to be monotone. This is useful for clients that want to
     /// use HarfBuzz as a cheap implementation of the Unicode Grapheme Cluster
     /// algorithm.
-    graphemes = c.HB_BUFFER_CLUSTER_LEVEL_GRAPHEMES,
+    graphemes = hb_buffer_cluster_level_graphemes,
 };
 
 /// The hb_glyph_info_t is the structure that holds information about the
